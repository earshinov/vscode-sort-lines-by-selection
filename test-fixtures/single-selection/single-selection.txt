e
cl[ass
f
a
g
der]ived
base
h
