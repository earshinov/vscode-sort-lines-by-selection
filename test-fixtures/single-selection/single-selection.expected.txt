e
a
cl[ass
der]ived
f
g
base
h
